### PR TITLE
Comparing ceph version

### PR DESCRIPTION
**Pull request by AmarnatReddy**
# Description
Comparing ceph version

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: ceph/ceph_admin/bootstrap.py [hunk 1]
```diff
@@ -364,7 +365,7 @@ def bootstrap(self: CephAdmProtocol, config: Dict):
                 "snmp_gateway",
                 "loki",
             ]
-            if _rhcs_version >= 7.0:
+            if self.cluster.rhcs_version >= LooseVersion("7.0"):
```
Comment (vamahaja) [inline ceph/ceph_admin/bootstrap.py:368]: why `self.cluster.rhcs_version` ? `_rhcs_version` passed through the upgrade suite which needs to be validated.

Comment (AmarnatReddy) [reply ceph/ceph_admin/bootstrap.py:368]: Based on the --rhbuild or rhcs_version it gets assigned to this variable ultimately
https://github.com/AmarnatReddy/cephci/blob/e947fcb256a797cba221d44b7299205b8a753871/ceph/ceph_admin/bootstrap.py#L224

Logs without RHCS Version : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-4ISGD8/

Comment (vamahaja) [reply ceph/ceph_admin/bootstrap.py:368]: @AmarnatReddy as per my understanding `_rhcs_version` is pre-upgrade version and `rhBuild` is post upgrade version. How could both same ?
Refer - https://github.com/red-hat-storage/cephci/blob/master/suites/reef/upgrades/tier-2-upgrade-rhcs-5x-to-7x-standby-mgr.yaml#L27

Comment (tintumathew10) [reply ceph/ceph_admin/bootstrap.py:368]: @vamahaja this code is doing it - self.cluster.rhcs_version = _rhcs_version or rhbuild

**Review by tintumathew10 — APPROVED** [review]
LGTM

File: ceph/ceph_admin/bootstrap.py (hunk 1 above)
Comment (AmarnatReddy) [reply ceph/ceph_admin/bootstrap.py:368]: While Cluster is getting created as part of test_cephadm test and this value will be either _rhcs_version(if provided in suite file) or rhbuild given in CLI arguments.
_rhcs_version will take priority

**Pull request merged**